app/src/main/java/com/android/streetworkapp/ui/profile/Profile.kt

```diff
@@ -105,6 +111,56 @@ fun ProfileScreen(
   }
 }
 
+/**
+ * This function displays the setting dialog. Called in mainActivity with TopAppBar actions
+ * callbacks
+ *
+ * @param showDialog - MutableState to show the dialog
+ * @param user - The user whose settings are to be displayed.
+ */
+@Composable
+fun SettingDialog(
+    showDialog: MutableState<Boolean>,
+    user: User? = null,
```

Comment:
user is not used in the function, is it needed for a future feature ?

Reply:
Do you think it is not important to have the user whose settings you want to edit ? Also again this PR is not ready for review, as you can see from the empy settings dialog.